### PR TITLE
SPEC: clarify language around the capabilities flags

### PR DESCRIPTION
**Pull request by philips**
This essentially offers two options:

- Remove certain capabilities and include all others
- Retain certain capabilities and removea all others

Fix everything up to be consistent with the remove-set and retain-set
language.

### CODE REVIEW

File: SPEC.md [hunk 1]
```diff
@@ -261,7 +261,7 @@ The first example is "capabilities" but this will be expanded to include things
 
 **Parameters:**
 
-* **set** list of capabilities that will be retained in the process's capabilities bounding set
+* **set** list of capabilities that will be retained in the process's capabilities bounding set, all others will be removed
```
Comment (jonboulle) [inline SPEC.md]: Add a trailing period for consistency with the other.

**Comment by jonboulle** [comment]
This is okay, but still underspecified. I would argue that _both_ methods are still somewhat implicit/non-deterministic; we need to make some kind of reference to what the "default" set would be.  I wouldn't mind addressing this via #336

**Comment by philips** [comment]
@jonboulle Setting up a default set is just going to get us into a world of trouble as the Kernel adds more overtime. I am fine with saying the image must explicitly give us the CAPS it wants and remove the remove-set.

**Comment by jonboulle** [comment]
@philips I didn't actually mean codifying a specific set in the spec, but at least saying that it would be in reference to the host kernel's default for example. Anyway I am still sympathetic to #388 , let's continue there.

**Comment by jonboulle** [comment]
Merging this as it fixes the schema/spec divergence, will follow up to fix capabilities more better.

**Pull request merged**